src/qt/forms/receivecoinsdialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>776</width>
-    <height>364</height>
+    <height>396</height>
```

Comment:
Whats the reason for changing the height?

Reply:
I guess it's QT Creators magic... I guess others do usually edit those files "by hand" to avoid such "larger" unnecessary changes.

Reply:
It was magic. Didn't seem worth tweaking with so many other changes in that file, but will keep in mind.